### PR TITLE
Add the rest of the LEGO games, as well as 'de Blob 2'

### PR DESCRIPTION
**Pull request by NoriLynnfield**
...Not sure why I threw dB2 in with the LEGO games, but it works

LEGO Games:
* Logos sourced from the 3DS version or Steam, except LEGO Rock Band which comes from [here](https://www.steamgriddb.com/logo/33381)
* Sounds sourced from the 3DS version if available, otherwise trimmed down versions of the Wii's banner used instead.

de Blob 2:
* Logo sourced from the Steam page
* Sound from the game itself (Level start music)

...If I messed anything up please tell me, it's 1:47 AM as I submit this

### CODE REVIEW

File: assets/B83/B83.png
Comment (lifehackerhansol) [inline assets/B83/B83.png:1]: this image needs fixing, there appears to be a rectangle border around the image and it's also not centered

**Review by lifehackerhansol — CHANGES_REQUESTED** [review]

**Comment by lifehackerhansol** [comment]
all the other ones seem ok

File: assets/B83/B83.png
Comment (NoriLynnfield) [reply assets/B83/B83.png:1]: I'll get on that right away

**Comment by NoriLynnfield** [comment]
Last minute addition of the game I nearly forgot existed

**Comment by lifehackerhansol** [comment]
last thing before I merge, can you just make a list of all the games we added here? all else seems fine now, I just like having a list of added things in the commit message instead of "see the commit data for what's inside"

**Comment by NoriLynnfield** [comment]
Sure thing, just give me a minute or three.

(Also, one of my comments has the word 'Pending' above it; I think you need to approve the changes first? I'm not very used to doing PRs honestly)

**Comment by lifehackerhansol** [comment]
> Sure thing, just give me a minute or three.
> 
> (Also, one of my comments has the word 'Pending' above it; I think you need to approve the changes first? I'm not very used to doing PRs honestly)

tbqh I haven't reviewed images before, idk how that looks

I do need to approve the PR (I usually do it)
github sucks lol

**Comment by NoriLynnfield** [comment]
One game list, coming right up!

* de Blob 2 [VD2]
* LEGO Star Wars II: The Original Trilogy [AL7]
* LEGO Star Wars III: The Clone Wars [BL9]
* LEGO Star Wars: The Complete Saga [YLG]
* LEGO Rock Band [BLR]
* LEGO Indiana Jones: The Original Adventures [YLJ]
* LEGO Indiana Jones 2: The Adventure Continues [BLJ]
* LEGO Batman: The Videogame [YJB]
* LEGO Batman 2: DC Super Heroes [B6F]
* LEGO Friends [TLJ]
* LEGO Harry Potter: Years 5-7 [B83]
* LEGO Legends of Chima: Laval's Journey [TCB]
* LEGO Marvel Super Heroes: Universe in Peril [TLM]
* LEGO Pirates of the Caribbean: The Video Game [BZD]
* LEGO The Lord of the Rings [TLR]

**Review by lifehackerhansol — APPROVED** [review]

**Comment by NoriLynnfield** [comment]
Seems like these buttons do something...

Screenshot 2023-04-17 023921
---
Screenshot 2023-04-17 023931
Screenshot 2023-04-17 023940

I appear to have... started a review?

**Comment by lifehackerhansol** [comment]
> Seems like these buttons do something...
> 
> ## Screenshot 2023-04-17 023921
> Screenshot 2023-04-17 023931 Screenshot 2023-04-17 023940
> 
> I appear to have... started a review?

ah. yes that's the case if you press "start a review"
normally, if someone else requested a change, then you can just "add comment" instead. think of a "review" as "you should probably change something here" and a comment as just extra notes or something

**Review by NoriLynnfield — COMMENTED** [review]
The word 'Pending' was nagging at me; I apologize if this messes something up, but let's hope it doesn't

Edit: Okay everything's fine now I can go to sleep

**Pull request merged**